tests/test_manaus/test_iam.py

```diff
@@ -0,0 +1,160 @@
+from datetime import datetime, timezone
+from unittest.mock import MagicMock
+
+from senza.manaus.iam import IAM, IAMServerCertificate
+
+IAM_CERT1 = {'CertificateBody': 'body',
+             'CertificateChain': 'chain',
+             'ServerCertificateMetadata': {
+                 'Arn': 'arn:aws:iam::0000:server-certificate/senza-example-com',
```

Comment:
You can't reuse a used Name for a new Certificate.

You must add a prefix like the upload date